projects/pastanaga-angular/src/lib/breakpoint-observer/breakpoint.observer.ts

```diff
@@ -3,14 +3,15 @@ import { fromEvent, Observable } from 'rxjs';
 import { distinctUntilChanged, shareReplay, startWith, map } from 'rxjs/operators';
 
 export enum ViewportSize {
-    small = 320,
+    small = 0,
     medium = 600,
-    large = 1024,
+    large = 1025,
```

Comment:
Shouldn't we use 1024? I think it's the standard.

Reply:
ipad is 1024px height, so 1024px width on landscape mode, I think that's why.
Here is the breakpoint list we have in atlas:
![image](https://user-images.githubusercontent.com/3293937/91999990-4c78e200-ed3d-11ea-97e0-f8ee9dd68f79.png)